git_machete/cli.py

```diff
@@ -532,21 +532,22 @@ def launch(orig_args: List[str]) -> None:
                     "rather than a regular file, aborting")
 
         if cmd == "add":
-            machete_client.read_definition_file()
+            machete_client.read_definition_file(perform_interactive_slide_out=perform_interactive_slide_out(cmd))
```

Comment:
Maybe extract this var to before the `if`, and just pass it everywhere like:
```suggestion
            machete_client.read_definition_file(perform_interactive_slide_out)
```